.github/workflows/build.yml

```diff
@@ -20,24 +23,36 @@ jobs:
           fetch-depth: 1
       - name: build base image
         run: |
-          cd image && ruby auto_build.rb base
+          cd image && ruby auto_build.rb ${{ matrix.name }}
       - name: build test_build image
+        if: matrix.name == 'base'
         run: |
           cd image && ruby auto_build.rb discourse_test_build
       - name: run specs
+        if: matrix.name == 'base'
```

Comment:
Does this mean that in the case of failing specs, it won't publish `base:release` but **will** go ahead with `base:slim`? 🤔

Reply:
Yes that is true. Let me see if I can make it install the dependencies and run the specs 👀